test/integrations/testUtils.ts

```diff
@@ -277,6 +277,7 @@ export const generateSimplifiedTrackPayload: any = (parametersOverride: any) =>
       device: parametersOverride.context.device,
       os: parametersOverride.context.os,
       app: parametersOverride.context.app,
+      page: parametersOverride.context.page,
```

Comment:
```suggestion
      ...(parametersOverride?.context?.page ?? {page: parametersOverride.context.page}),
```
suggestion to make sure nothing else gets affected

Reply:
Tried adding it, but then it's not able to find context.page.